src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java

```diff
@@ -148,6 +148,23 @@ public boolean hasClaimantAskedToSignSettlementAgreement() {
         return false;
     }
 
+    public boolean hasClaimantAndDefendantSignedSettlementAgreement() {
```

Comment:
Move this to a common service instead of duplicating the same methods on 2 sites.